pages/linux/pacdiff.md

```diff
@@ -0,0 +1,28 @@
+# pacdiff
+
+> Pacorig, pacnew and pacsave maintenance utility for pacman.
+> More information: <https://manned.org/pacdiff>.
+
+- Look for files that need maintenance:
+
+`pacdiff`
+
+- Use sudo and sudoedit to remove and merge files:
+
+`pacdiff --sudo`
+
+- Store the original version of any file you `(O)verwrite with` as a `.bak`:
+
+`pacdiff -s --backup`
+
+- Override the program for `(M)erge`ing changes (`vim -d` is default):
```

Comment:
```suggestion
- Use a specific editor to view and merge conflicted files:
```

Reply:
@navarroaxel is it not important to clarify the default? Do you think it's valuable to quote the exact prompt of "(M)erge" instead of paraphrasing?

Reply:
you're right!

Reply:
It should be present tense, "conflicting" right? Do you think the exact prompt should be included?

Reply:
which prompt ?

Reply:
When you run the command, you get prompted repeatedly to decide what to do with the given file -- I felt it was more clear to include the exact option of "(M)erge" to clarify when the variable matters. The full prompt is
```bash
ask "(V)iew, (M)erge, (S)kip, (R)emove %s, (O)verwrite with %s, (Q)uit: [v/m/s/r/o/q] " "$file_type" "$file_type"
```
I don't know if `DIFFPROG` is also used for `(V)iew`ing? I can't remember... `pacdiff` help says:
```
Environment Variables:
  DIFFPROG        override the merge program (default: 'vim -d')
 ```
 Checking the source code, and V calls view_diff, and M calls merge_file...
Both of those functions use `$diffprog`, so it is used for both viewing and merging files.

Would it be acceptable to update this to:
> - Use a specific editor to `(V)iew` and `(M)erge` configuration files (default is `vim -d`):

or is this unnecessarily clear?

Reply:
IMHO the prompt is pretty clear. I focus the example description on which review process the example starts and not how to interact with the `pacdiff` script itself.